src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageReactions/MessageReactions.styles.ts

```diff
@@ -46,11 +46,19 @@ export const messageReactionWrapper: CSSObject = {
   maxWidth: '100%',
   marginRight: 'var(--conversation-message-timestamp-width)',
   '.tooltip-content': {
-    backgroundColor: 'var(--gray-95) !important',
+    backgroundColor: 'var(--white) !important',
```

Comment:
isn't it supposed to change according to dark mode? 🤔

Reply:
How was it implemented before? It used to be good on both dark and light mode, right?